pairinteraction/CMakeLists.txt

```diff
@@ -360,14 +364,23 @@ if( WITH_DMG )
 elseif ( NOT WIN32 )
 
   if(WITH_PYTHON)
-    # Hopefully better in the future... https://gitlab.kitware.com/cmake/cmake/-/issues/24213
-    set(PY_MOD_DIR "lib/python${Python3_VERSION_MAJOR}.${Python3_VERSION_MINOR}/site-packages/pairinteraction")
+    if(SKBUILD)
+      set(PY_MOD_DIR "${SKBUILD_PLATLIB_DIR}/pairinteraction")
+      install(TARGETS pairinteraction LIBRARY DESTINATION ${PY_MOD_DIR})
+    else()
+      # Hopefully better in the future... https://gitlab.kitware.com/cmake/cmake/-/issues/24213
+      set(PY_MOD_DIR "lib/python${Python3_VERSION_MAJOR}.${Python3_VERSION_MINOR}/site-packages/pairinteraction")
+    endif()
+
     install(TARGETS binding LIBRARY DESTINATION ${PY_MOD_DIR})
-    install(FILES ${CMAKE_CURRENT_BINARY_DIR}/binding.py DESTINATION ${PY_MOD_DIR})
+    install(TARGETS pairinteraction LIBRARY DESTINATION ${PY_MOD_DIR})
+    install(TARGETS pairinteraction-backend-deprecated RUNTIME DESTINATION ${PY_MOD_DIR})
+    install(FILES ${SWIG_MODULE_binding_SUPPORT_FILES} DESTINATION ${PY_MOD_DIR})
     install(FILES ${CMAKE_CURRENT_BINARY_DIR}/pireal.py DESTINATION ${PY_MOD_DIR})
     install(FILES ${CMAKE_CURRENT_BINARY_DIR}/picomplex.py DESTINATION ${PY_MOD_DIR})
```

Comment:
Figure out a better way to install stuff and also which are the correct directories inside the wheel.